modules/manage/partials/monitor-redpanda.adoc

```diff
@@ -281,11 +281,15 @@ To detect unexpected idling, you can query the rate of change as a percentage of
 rate(redpanda_cpu_busy_seconds_total[5m])
 ----
 
-[NOTE]
+[TIP]
 ====
-While CPU utilization at the host-level can typically appear high (for example, 99-100% utilization) when I/O events like message arrival occur, the actual Redpanda process utilization is likely low and can be monitored with `redpanda_cpu_busy_seconds_total`.
+While CPU utilization at the host-level can typically appear high (for example, 99-100% utilization) when I/O events like message arrival occur, the actual Redpanda process utilization is likely low, in contrast to what system-level metrics might indicate such as those provided by the `top` command. 
+
+This high host-level CPU utilization happens because Redpanda uses Seastar, which runs event loops on every core (also referred to as a _reactor_), constantly polling for the next task. This process never blocks and will increment clock ticks. It doesn't necessarily mean that Redpanda is busy.
+
+Use `redpanda_cpu_busy_seconds_total` (public metrics) or xref:reference:internal-metrics-reference.adoc#vectorized_reactor_utilization[`vectorized_reactor_utilization`] (internal metrics) to monitor the actual Redpanda CPU utilization. When these particular metrics approach 100% utilization, you will then also start to see higher produce and consume <<latency,latency>>.
 
-This high host-level CPU utilization happens because Redpanda uses Seastar and constantly polls and increments clock ticks, and it doesn't necessarily mean that Redpanda is busy.
+If achieving very low latency consume is not a priority, consider tuning the xref:reference:tunable-properties.adoc#fetch_reads_debounce_timeout[`fetch_reads_debounce_timeout`] property. The default setting at 1ms means that Redpanda pings the partition once per millisecond of the given client poll cycle. Setting the duration at a higher value, for example up to 10ms, can help reduce CPU utilization.
```

Comment:
I don't think we should include this property at all. We don't want people to set properties which they don't really understand. This should be reserved for CS etc to apply it in a targeted fashion where needed. 

We do now finally have a PR up which should make this unneeded and it looks likely that we can also backport this fix.

One of our general principles is that Redpanda should not require any changing of the tunables and they are kind of there as an escape hatch if things go wrong.

Reply:
@hcoyote If it sounds good, I'll remove this paragraph. We could replace it with other info too, if there are any other actions we might recommend to users?

Reply:
@StephanDollberg  this goes back to our discussion about what the appropriate value is for fetch_reads_debounce_timeout out of the box. With cloud deploys, it's 5ms. With self-managed it's 1. And we've run into several cases already where 1 is probably overly aggressive on self-hosted. 

If we do follow through with the plan to align the defaults between self-managed/cloud, then I'm ok removing this (or maybe moving this to a deeper/nuanced discussion in a KB). But I think it's important for users to understand the mechanism here, as we constantly run into the "why does redpanda take so much cpu" question.

Reply:
> If we do follow through with the plan to align the defaults between self-managed/cloud

Like I said above, I am positive that hopefully soon this issue resolved. 

> as we constantly run into the "why does redpanda take so much cpu" question.

I understand but that issue is way larger than just fetch debounce. 

I'd be happy with a more KB/known-issue style article (that we can easily deprecate/remove later) about this than just a blank recommendation.

Reply:
If we're backporting the fix that makes tuning it totally unnecessary, I would err on the side of not documenting this (at least not in the vanilla 'monitoring redpanda' page).  Perhaps in the config reference for tunables (which targets more advanced users), but the monitoring page at least should be for things that are mainstream